cmake/BuildLibElf.cmake

```diff
@@ -0,0 +1,15 @@
+include(ExternalProject)
+
+ExternalProject_Add(libelf
+  PREFIX "vendor/elfutils"
+  URL http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_0.163.orig.tar.bz2
```

Comment:
...Alternatively, I could check in `elfutils_0.163.orig.tar.bz2` on the repository. It's a tad unorthodox, but I know of a lot of people who do just that for upstream dependencies.